packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -57,30 +58,56 @@ function getCellStyle(
   displayValue: DisplayValue,
   disableOverflowOnHover = false
 ) {
-  if (field.config.custom?.displayMode === TableCellDisplayMode.ColorText) {
-    return tableStyles.buildCellContainerStyle(displayValue.color, undefined, !disableOverflowOnHover);
+  // How much to darken elements depends upon if we're in dark mode
+  const darkeningFactor = tableStyles.theme.isDark ? 1 : -0.7;
+
+  // See if we're using deprecated settings
+  const usingDeprecatedSettings = field.config.custom.cellOptions.backgroundDisplayMode === undefined;
```

Comment:
deprecated?

Reply:
ah, I see now it checks if it's undefined, sorry

Reply:
maybe it's easier to understand this logic if it checks if the old settings exists, rather than checking if the new are undefined

Reply:
Updated to check for presence of old `displayMode`